go/dbproxy/local_table.go

```diff
@@ -110,7 +110,7 @@ func localTableDeleteOne(c *gin.Context) {
 		}
 		return
 	}
-	c.JSON(http.StatusOK, gin.H{"data": "ok"})
+	c.JSON(http.StatusOK, gin.H{"data": true})
```

Comment:
In other APIs, we use `{"result":"ok", "data": ...}`. It's best to be consistent I think.